massa-final-state/src/executed_ops.rs

```diff
@@ -219,17 +249,24 @@ impl ExecutedOpsSerializer {
 impl Serializer<ExecutedOps> for ExecutedOpsSerializer {
     fn serialize(&self, value: &ExecutedOps, buffer: &mut Vec<u8>) -> Result<(), SerializeError> {
         // encode the number of entries
-        let entry_count: u64 = value.0.len().try_into().map_err(|err| {
+        let entry_count: u64 = value.ops.len().try_into().map_err(|err| {
             SerializeError::GeneralError(format!("too many entries in ExecutedOps: {}", err))
         })?;
         self.u64_serializer.serialize(&entry_count, buffer)?;
 
         // encode entries
-        for (op_id, slot) in &value.0 {
+        for (op_id, slot) in &value.ops {
             buffer.extend(op_id.to_bytes());
             self.slot_serializer.serialize(slot, buffer)?;
         }
 
+        // note: mimic the behaviour of OptionSerializer because we have not hash serializer
```

Comment:
maybe open an issue to make a hash serializer.

Reply:
Implemented the hash serializer in the end.